pages/[name]/[id].js

```diff
@@ -16,11 +17,21 @@ export async function getStaticProps() {
     images: prod.images,
     description: prod.description,
     name: prod.name,
-    price: "",
+    price: '',
   }));
-  productsWithPrices.forEach((prod) => {
-prod.price = 20;
-  });
+
+  function integratePrice() {
```

Comment:
lets use `.map` in here and delete the `.map` on lines 15-20. 

The way you're doing the `for` loop is basically baking your own `.map`, except you're _mutating_ the array. `.map` is to be used because it returns a new array.